index.html

```diff
@@ -12,7 +12,7 @@
 		/>
 		<link rel="icon" type="image/svg+xml" href="/src/favicon.ico" />
 		<meta name="theme-color" content="#000000" />
-		<title>Smart Shopping List</title>
+		<title>Shop-Ade smart shopping application</title>
```

Comment:
I really love the favicon! Super rad! Might I make a suggestion for the title? 'Shop-ade - Your Smart Shopping App' or if sticking with what's already there, just capitalizing everything so it says 'Shop-Ade Smart Shopping App'. Otherwise, great job, going to approve!

Reply:
Agree! The favicon is perfect. Please update the title so that it is capitalized: I am fine with either 'Shop-Ade Smart Shopping App', 'Shop-Ade - Your Smart Shopping App', or even just 'Shop-Ade' (nice and short).